src/Steps/TMSSReqs.tsx

```diff
@@ -109,20 +113,29 @@ export const TMSSReqs = () => {
         { name: "Non-nuclear Munitions Basic Information", id: "tmss_7_c" },
         { name: "Non-nuclear Loading Procedures", id: "tmss_7_d" },
         {
-          name: "Non-nuclear Munitions Loading Standard Data Packages (SDPs)",
+          name: "Integrated Combat Turnaround Procedures Manual",
           id: "tmss_7_e",
         },
-        { name: "Loading Procedures Checklist", id: "tmss_7_f" },
-        { name: "Single Loading Procedures Checklist", id: "tmss_7_g" },
-        { name: "Integrated Loading Procedures Checklists", id: "tmss_7_h" },
-        { name: "Family Group Loading Procedures Checklist", id: "tmss_7_i" },
+        {
+          name: "Non-nuclear Munitions Loading Standard Data Packages (SDPs)",
+          id: "tmss_7_f",
+        },
+        { name: "Loading Procedures Checklist", id: "tmss_7_g" },
+        { name: "Single Loading Procedures Checklist", id: "tmss_7_h" },
+        { name: "Integrated Loading Procedures Checklists", id: "tmss_7_i" },
+        { name: "Family Group Loading Procedures Checklist", id: "tmss_7_j" },
         {
           name: "Nuclear Weapons Loading Procedure Checklists",
-          id: "tmss_7_j",
+          id: "tmss_7_k",
+        },
+        { name: "Functional Check Procedures Checklist", id: "tmss_7_l" },
+        {
+          name: "Integrated Combat Turnaround Procedures Checklist",
+          id: "tmss_7_m",
         },
-        { name: "Functional Check Procedures Checklist", id: "tmss_7_k" },
-        { name: "NATO Stage B Cross-Servicing Checklists", id: "tmss_7_l" },
-        { name: "End of Runway (EOR) Procedures Checklist", id: "tmss_7_m" },
+        { name: "NATO Stage B Cross-Servicing Checklists", id: "tmss_7_n" },
+        { name: "End of Runway (EOR) Procedures Checklist", id: "tmss_7_o" },
+        { name: "Smart Weapons Load Checklist", id: "tmss_7_p" },
```

Comment:
Check the Word doc on this one. Looks like there is a visible table line above it that shouldn't be there.